app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  validates_presence_of :body, :receiver_id
```

Comment:
it would be nice to add `user_id` also.

Reply:
I think it would be much better in understanding of models if the relation `user` was `author`